op-node/rollup/derive/kromampt_upgrade_transactions.go

```diff
@@ -240,7 +240,7 @@ func KromaMPTNetworkUpgradeTransactions(chainID *big.Int) ([]hexutil.Bytes, erro
 		To:                  nil,
 		Mint:                big.NewInt(0),
 		Value:               big.NewInt(0),
-		Gas:                 1_000_000,
+		Gas:                 1_500_000,
```

Comment:
Just a question, I'm curious if the gas limit was short than actual gas usage, how the migration_test passed?

Reply:
Did you mean migration_test? It seems that there is no checking logic for upgraded GPOs in the test, even if the deployment fails for the GPOs.

Reply:
This case is covered at PR #404, it checks if all kroma mpt upgrade txs succeed.